tests/unit/amo/components/TestHeader.js

```diff
@@ -185,7 +185,7 @@ describe(__filename, () => {
       userProps: { username },
     });
     const wrapper = renderHeader({ store });
-    const link = wrapper.find('.Header-user-menu-developers-submissions-on-link');
+    const link = wrapper.find('[className = "Header-user-menu-developers-submissions-on-link"]');
```

Comment:
The test case was correct. Look at your source file and see if you can find `.Header-user-menu-developers-submissions-on-link` 😉

Reply:
Oops, i think no :/

Reply:
```.Header-user-menu-developers-submissions-on-link``` it does not exist :stuck_out_tongue:

Reply:
You should add this class name to the `Link` you have added, but in the source file.

`wrapper.find()` tries to find components into the source component (the component/file under test). Your test case is correct, because it tries to find the link you have added. The problem is that you look for a component with a given class name (`.Header-user-menu-developers-submissions-on-link`), which is good. But, because you don't specify it in the source file, the test case fails because it does not find anything, like you :)

So, add this class name to the link you have added in the source file.

Reply:
@willdurand got it , thanks for the help :heart:  .